java/lib/core/src/main/java/org/eclipse/tahu/json/PropertySetDeserializer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Licensed Materials - Property of Cirrus Link Solutions
+ * Copyright (c) 2023 Cirrus Link Solutions LLC - All Rights Reserved
+ * Unauthorized copying of this file, via any medium is strictly prohibited
+ * Proprietary and confidential
+ */
+package org.eclipse.tahu.json;
```

Comment:
This is Tahu - so you need a different statement here.

Reply:
Fixed, thank you!